Map Version Edits.R

```diff
@@ -71,21 +92,18 @@ network <- suppressWarnings({
 })
 
 
-# Assuming 'network' is your sfnetwork
```

Comment:
This comment didn't seem like it was adding anything meaningful, so I removed it.